tests/manifests/dvImageio.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: "test-dv"
+spec:
+  source:
+      imageio:
+         url: "http://f24:8080/ovirt-engine/api"
+         secretRef: "secret"
+         certConfigMap: "tls-certs"
```

Comment:
Maybe add a fake cert configmap yaml to show the format.

Reply:
Format is the same as othe other configmaps so linking to existing configmap should be sufficient.

Reply:
Added in #1151